python/ray/worker.py

```diff
@@ -1259,15 +1244,10 @@ def connect(node,
         node.redis_password,
     )
     worker.core_worker = ray._raylet.CoreWorker(
-        (mode == SCRIPT_MODE),
-        node.plasma_store_socket_name,
-        node.raylet_socket_name,
-        job_id,
-        gcs_options,
-        node.get_logs_dir_path(),
-        node.node_ip_address,
-        node.node_manager_port,
-    )
+        (mode == SCRIPT_MODE), node.plasma_store_socket_name,
+        node.raylet_socket_name, job_id, gcs_options, node.get_logs_dir_path(),
+        node.node_ip_address, node.node_manager_port, driver_name,
+        log_stdout_file_name, log_stderr_file_name)
```

Comment:
can you keep this one item per line? (just add a comma after the last argument)

Reply:
Done.